optimize/backend/pom.xml

```diff
@@ -699,29 +660,23 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-collections4</artifactId>
       <version>4.4</version>
-      <scope>compile</scope>
     </dependency>
 
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-webmvc</artifactId>
-      <version>${spring.version}</version>
-      <scope>compile</scope>
     </dependency>
 
     <!-- scope here is wider than test. Dependency:analyze error suppressed -->
     <dependency>
       <groupId>commons-collections</groupId>
       <artifactId>commons-collections</artifactId>
-      <version>20040616</version>
-      <scope>compile</scope>
+      <version>3.2.2</version>
```

Comment:
❓ The Single Jar/App Stack uses `commons-collections4`. Could the usage of `commons-collections` be removed and replaced?

Reply:
I could find one single use of commons-collections. I replaced it using commons-collections4. This should allow me to remove the old library.